aerogear/keycloak-operator
# adding keycloak realm crd

## Motivation
https://trello.com/c/Cyp2anQA/198-5-add-basic-installation-operator-for-rh-sso

## What
Allow Realms to be provisioned seperate to keycloak instances

## Why
So that users that cannot provision keycloak can still use a realm in keycloak

## How
Keycloak and Keycloak realms were split into separate CRDs and reconciled seperately

## Verification Steps
Provision a cluster from: https://github.com/integr8ly/installation/pull/123

You should be able to login to the cluster using: admin@example.com / Password1
You should be able to login to 3scale using: evals01@example.com / Password1

## Checklist:


## Progress

- [x] Split keycloak realm out of keycloak CRD
